config/dev-local.js

```diff
@@ -28,7 +26,7 @@ module.exports = merge(base, devVm, {
 		},
 	},
 	server: {
-		graphqlUri: 'https://marketplace-api.dk1.kiva.org/graphql',
+		graphqlUri: 'https://gateway.development.kiva.org/graphql',
 		sessionUri: 'https://www.dev.kiva.org/start-ui-session',
 		// Alternate session url
 		// sessionUri: 'https://www.development.kiva.org/start-ui-session',
```

Comment:
Please go ahead and swap in this one and delete the other session uri as it's no longer available.